Parsers/ASimRegistryEvent/Parsers/ASimRegistryEventMicrosoftSysmon.yaml

```diff
@@ -93,76 +93,16 @@ ParserQuery: |
-                  EventCount = int(1), 
-                  EventVendor = "Microsoft",
-                  EventSchemaVersion = "0.1.0", 
-                  EventProduct = "Sysmon",
-                  EventOriginalType = tostring(EventID),
-                  EventType = tostring(EventData.EventType),
-                  DvcOs = "Windows",
-                  EventMessage = tostring(EventData.RenderedDescription), 
-                  ActorUsername = tostring(EventData.User),
-                  ActingProcessId = tostring(EventData.ProcessId),
-                  ActingProcessGuid = extract ('^{(.*)}$', 1, tostring(EventData.ProcessGuid), typeof(string)),
-                  ActingProcessName = tostring(EventData.Image),
-                  TargetObject = tostring(EventData.TargetObject),
-                  Parameter = tostring(EventData.Parameter)
-              | project-rename
-                  DvcHostName = Computer 
-              | lookup RegistryAction on EventType
-              | project-rename EventOriginalSubType = EventType
-              | project-rename EventType = NewEventType
-              // Normalize Key Hive
-              | parse TargetObject with KeyPrefix "\\" KeyMain
-              | lookup Hives on KeyPrefix
-              | extend Key = strcat (Hive, "\\", KeyMain)
-              | parse Parameter with KeyPrefix "\\" KeyMain
-              | lookup Hives on KeyPrefix
-              | extend NewName = strcat (Hive, "\\", KeyMain)
-              | project-away KeyPrefix, KeyMain, Hive
-              // Split Key and Value for relevant events 
-              | extend ParsedKey = extract_all (@"^(.+)\\(.+)$", Key)
-              | extend Key = iff (EventType in ("RegistryValueSet", "RegistryValueDeleted"), ParsedKey[0][0], Key)
-              | extend Value = iff (EventType in ("RegistryValueSet", "RegistryValueDeleted"), ParsedKey[0][1], "")
-              | extend ParsedKey = extract_all (@"^(.+)\\(.+)$", NewName)
-              | extend NewKey = ParsedKey[0][0]
-              | extend NewValue = ParsedKey[0][1]
-              | project-away ParsedKey, TargetObject, NewName
-              // Set normalized registry fields
               | extend
                   RegistryKey = iff (EventType == "RegistryKeyRenamed", NewKey, Key),
                   RegistryKeyModified = iff (EventType in ("RegistryKeyRenamed", "RegistryValueSet"), Key, ""),
                   RegistryValue = iff (EventType in ("RegistryValueSet", "RegistryValueDeleted"), Value, ""),
                   RegistryValueModified = iff (EventType == "RegistryValueSet", Value, ""),
                   RegistryValueData = iff (EventType == "RegistryValueSet", Parameter, ""),
-                  ActorUsernameType =  iff(isnotempty(ActorUsername), 'Windows', '')
+                  EventResult = "Success",
+                  EventSchema = "RegistryEvent",
+                  Rule=RuleName,
+                  ParentProcessId=ActingProcessId
```

Comment:
how we concluded this?

Reply:
checked in document,other parsers and tested in workspace

Reply:
didn't map ParentProcessId as value not found in source